### PR TITLE
Update draft-cds-rats-intel-corim-profile.md

### PR DESCRIPTION
**Pull request by nedmsmith**
Updates references, intro, background prose and fixes paragraphs with sentences broken by returns. Various other wording changes.

### CODE REVIEW

File: draft-cds-rats-intel-corim-profile.md [hunk 1]
```diff
@@ -200,13 +207,10 @@ The profile identifier for this profile is the OID:
...
 
-# CoMID Schema Extensions {#sec-comid-schema-extensions}
-
-The baseline CoMID schema for Reference Values is extended with an attribute that informs a Verifier as to which matching
-semantics to apply, whether they are equivalance, range, or set membership semantics.
+# CoMID Extensions {#sec-comid-extensions}
 
-This profile extends `measurement-values-map` with additional measurements that are used by Evidence,
-Reference Values, and Endorsed Values.
+This profile extencs the baseline CoMID for Reference Values with an expression that informs Verifiers about non-exact-match matching semantics that include: ranges, sets, masks.
```
Comment (fchinchilla) [inline draft-cds-rats-intel-corim-profile.md:212]: spelling of extends

Comment (nedmsmith) [reply draft-cds-rats-intel-corim-profile.md:212]: I'm confused. Is `extencs` a word?

Comment (nedmsmith) [reply draft-cds-rats-intel-corim-profile.md:212]: I agree with the rest of the change.

Comment (nedmsmith) [inline draft-cds-rats-intel-corim-profile.md:212]:
```suggestion
This profile extends the baseline CoMID for Reference Values with an expression that informs Verifiers about non-exact-match matching semantics that include: ranges, sets, and masks.
```

**Review by fchinchilla — APPROVED** [review]

**Review by jdbeaney — APPROVED** [review]

**Pull request merged**